docs/openapi.yaml

```diff
@@ -552,62 +668,132 @@ paths:
+          value: covid, threat
       schema:
-        $ref: "#/components/schemas/CategoryEnum"
-    categoryShouldParam:
-      name: category_should
+        type: string
+    abstractShouldParam:
+      name: abstract_should
       in: query
-      description: "Returns documents with these category codes. Support list of Enums, seperated by comma (+)."
-      example: "JournalArticle"
+      description: "Returns document with these words in the abstract."
       required: false
+      examples:
+        documents with all words:
+          value: covid threat
+        documents with either word:
+          value: covid, threat
       schema:
-        $ref: "#/components/schemas/CategoryEnum"
-    contributorIdParam:
-      name: contributor_id
+        type: string
+    aggregationParam:
+      name: aggregation
       in: query
-      description: "person id (URI)."
+      description: "Aggregations to include in the search."
+      required: false
+      examples:
+        all aggregations:
+          value: all
+        separate keys aggregations:
+          value: [ associatedArtifact, entityDescription, status, scientificIndex ]
+        one key aggregations:
+          value: "fundingSource, status, topLevelOrganisations, scientificIndex"
+      schema:
+        $ref: '#/components/schemas/AggregationNames'
+    contextTypeNotParam:
+      name: context_type_not
+      in: query
+      description: "Returns document without these contextTypes."
+      required: false
+      example: "Anthology"
+      schema:
+        $ref: '#/components/schemas/ContextValues'
+    contextTypeParam:
+      name: context_type
+      in: query
+      description: "Returns document with these contextTypes."
+      required: false
+      example: "Degree"
+      schema:
+        $ref: '#/components/schemas/ContextValues'
+    contextTypeShouldParam:
+      name: context_type_should
+      in: query
+      description: "Returns document with these contextTypes."
+      required: false
+      example: Book
+      schema:
+        $ref: '#/components/schemas/ContextValues'
+    contributorNameNotParam:
+      name: contributor_name_not
+      in: query
+      description: "Returns document without Author's name."
+      example: "John Doe"
+      required: false
+      schema:
+        type: string
+    contributorNameParam:
+      name: contributor_name
+      in: query
+      description: "Returns document with Author's name."
+      example: "John Doe"
+      required: false
+      schema:
+        type: string
+    contributorNameShouldParam:
+      name: contributor_name_should
+      in: query
+      description: "Returns document with similar Author names."
+      example: "John Doe"
       required: false
       schema:
         type: string
     contributorParam:
       name: contributor
       in: query
-      description: "returns document with Author's name or person id(URI)."
+      description: "Returns document with Author's id(URI)."
```

Comment:
Is a contributor an author? I have no idea, am just curious.

Reply:
yeah, kind of (co-author).